dbms/src/Flash/Coprocessor/DAGExpressionAnalyzer.cpp

```diff
@@ -192,11 +198,200 @@ void appendWindowDescription(
     window_columns.emplace_back(func_string, result_type);
     source_columns.emplace_back(func_string, result_type);
 }
+
+Window::ColumnType getColumnType(const ColumnPtr & col_ptr)
```

Comment:
If use TypeIndex, the code can be simplify like `col_ptr->getDataTypePtr()->getTypeIndex()`

Reply:
> getTypeIndex

okk, I will use `DataType`